### DISCUSSION TITLE
How to change the max upload size?

### DISCUSSION THREAD

**Discussion by KevMoore88** [mediacms-io/mediacms#1035] [Q&A]
I'm trying to upload some videos and I keep getting this error. I looked around at some of the config and .py files, bu nothing stood out to me that created a max upload size. Where and how can I change the size from 4.1GB to 10GB? This is a non-docker, single server build, the latest version as of today.

Thanks

<img width="532" alt="Screenshot 2024-07-01 at 08 59 59" src="https://github.com/mediacms-io/mediacms/assets/173363392/ed88abfa-3a75-47e6-aa0f-a9f3899d1977">

**Comment by zolbarna** [comment]
Find this setting in the cms/settings.py, and modify it:

```
# bytes, size of uploaded media
UPLOAD_MAX_SIZE = 800 * 1024 * 1000 * 5 
```
